ts/Series/MapBubble/MapBubbleSeries.ts

```diff
@@ -248,6 +250,8 @@ extend(MapBubbleSeries.prototype, {
 
     processData: MapSeries.prototype.processData,
 
+    searchPoint: noop as any, // #16896
```

Comment:
What about other series types (not only a `mapbubble`)? As far as I know, `tooltip.shared` doesn't work for all the Highcharts Maps series, so maybe we can remove this function from the whole Maps product (`map` series, the rest inherits from `map`)?

Reply:
It looks like at least the `map` series is covered (it has `searchPoint: noop`.

However, it may be that we actually want to use k-d-search with the MapBubble. So I propose not setting it to noop but rather implement it:

```ts
    public searchPoint(e: PointerEvent, compareX?: boolean): Point | undefined {
        return this.searchKDTree({
            clientX: e.chartX - this.chart.plotLeft,
            plotY: e.chartY - this.chart.plotTop
        }, compareX, e);
    }
```